components/atom/button/README.md

```diff
@@ -0,0 +1,62 @@
+# Button
+
+> Atom Element: SUI button
+
+![](./assets/screenshot.png)
+
+## Installation
+
+```sh
+npm install @schibstedspain/sui-atom-button --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import Button from '@schibstedspain/sui-atom-button'
+
+return (<div>
+  <Button>Normal</Button>
+  <Button focused>Focused</Button>
+  <Button disabled>Disabled</Button>
+</div>)
+
+```
+
+### Flexible props
+
+All props available from regular buttons an be used.
+
+```js
+import Button from '@schibstedspain/sui-atom-button'
+
+return (<div>
+  <AtomButton onClick={() => alert('Primary with onClick')}>
```

Comment:
does make sense to add the context in the components? IMHO i would use only the component name.

Reply:
What do you mean? I don't understand.

Reply:
i speak about the component name. would be a better option to name it as `Button` instead of `AtomButton`?

Reply:
It's a mistake of the readme...
I'll change it to Button here. But in the index.js, the displayName will remain beting AtomButton. Agree?